docs/src/modules/java/pages/agents.adoc

```diff
@@ -679,6 +679,81 @@ include::example$multi-agent/src/main/java/demo/multiagent/application/AgentTeam
 
 You find the full source code for this multi-agent sample in the link:https://github.com/akka-samples/multi-agent[akka-samples/multi-agent GitHub Repository].
 
+== Guardrails
+
+Guardrails can protect against harmful inputs, such as jailbreak attempts, and damaging output, such as mentions of a competitor’s product.
+
+A specific guardrail implements the link:_attachments/api/akka/javasdk/agent/Guardrail.html[`Guardrail` interface]. It takes the input or output text as a parameter and a result if it passed the validation or not, including an explanation of why the decision was made. These results are included in metrics and traces. A guardrail can abort the interaction with the model, or only report the problem and continue anyway.
+
+Guardrails are enabled by configuration, to be able to enforce at an enterprise level that certain guardrails are always used.
```

Comment:
What does "enterprise level" mean? Should it be stated in Akka terms instead  "project" and/or "service", or explained at that level in addition to "enterprise"?